pkg/credentialprovider/azure/azure_acr_helper.go

```diff
@@ -48,12 +48,15 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"strconv"
 	"strings"
 	"unicode"
+
+	"k8s.io/kubernetes/pkg/credentialprovider"
```

Comment:
I wouldn't add a package dependency just to read a 1MB constant. Define it locally if needed.

Reply:
I think an interesting question is whether its important that the azure_acr_helper uses the same MaxReadLength as the code in config.go reading from docker/config.json. If no, I agree with @liggitt to define a local constant. If it is important, than I worry that defining another separate constant would create the possibility for these constants to diverge.

Reply:
I think an interesting question is whether its important that the azure_acr_helper uses the same MaxReadLength as the code in config.go reading from docker/config.json. If no, I agree with @liggitt to define a local constant. If it is important, than I worry that defining another separate constant would create the possibility for these constants to diverge.

Reply:
Removed this package dependency. Thanks!